plenum/test/view_change_with_delays/test_view_change_with_different_prepare_certificate.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+from plenum.common.constants import PREPREPARE
+from plenum.common.messages.node_messages import ThreePhaseKey
+from plenum.test.delayers import ppDelay, msg_rep_delay
+from plenum.test.helper import sdk_send_random_and_check, \
+    sdk_send_random_request
+from plenum.test.stasher import delay_rules
+from plenum.test.test_node import ensureElectionsDone
+from stp_core.loop.eventually import eventually
+
+
+def check_prepare_certificate(nodes, ppSeqNo):
+    for node in nodes:
+        key = (node.viewNo, ppSeqNo)
+        quorum = node.master_replica.quorums.prepare.value
+        assert node.master_replica.prepares.hasQuorum(ThreePhaseKey(*key),
+                                                      quorum)
+
+
+def test_view_change_with_different_prepare_certificate(looper, txnPoolNodeSet,
```

Comment:
Why don't we just create a simple unit test that just adds a quorum of Prepares from other nodes, but doesn't add PrePrepare?
See `test_lst_sertificate_return_max_3PC_key_of_quorumed_prepare` as an example.

Reply:
Because we want to check that the prepare does not get in replica.prepares without preprepare. It will be more useful with an integration tests.